lib/src/grpc_api_impl/grpc.dart

```diff
@@ -295,6 +296,14 @@ class Client {
 class Dialer {
   static const Duration MinimumTimeBetweenConnects = const Duration(seconds: 2);
 
+  // We delay the completion slightly for our http/2 client to receive
+  // the server settings:  If there are many RPCs pending, then our
+  // client will immediatly send all of the RPCs to the connection,
+  // possibly running over the max-concurrent-streams setting the server
+  // is sending to the client (which will cause the server to terminate
+  // the connection).
+  static const Duration estimatedRTT = const Duration(milliseconds: 20);
```

Comment:
Can be private?

Reply:
Done.

Reply:
It can, though this library is not exposed to users, so it makes little difference.